.github/workflows/test-cartesian.yml

```diff
@@ -53,8 +49,11 @@ jobs:
     - name: Run CPU 'cartesian' tests with nox
       env:
         NUM_PROCESSES: auto
+        CI_NOX_PREFIX: GT4PY
+        GT4PY_CI_NOX_RUN_ONLY_IF_CHANGED_FROM: "origin/${{ github.base_ref }}"
+        GT4PY_CI_NOX_VERBOSE: 1
         # Force a specific compiler for setuptools/distutils
         CXX: g++
         CC: gcc
       shell: bash
-      run: uv run nox -s 'test_cartesian-${{ matrix.python-version }}(${{ matrix.codegen-factor }}, cpu)'
+      run: uv run --script noxfile.py -s 'test_cartesian-${{ matrix.python-version }}(${{ matrix.codegen-factor }}, cpu)'
```

Comment:
Do you need to change to running nox as script in order to be able to customize the session? Or is this barely because of the sys.path issue?